WD-Firmware/examples/demo/rtosal_demo.c

```diff
@@ -0,0 +1,459 @@
+/*
+* SPDX-License-Identifier: Apache-2.0

```

Comment:
check license with origin main.c from SiFive

Reply:
Is it required anymore?